calplus/tests/fake_config_file.conf

```diff
@@ -34,6 +35,7 @@ os_identity_api_version = 3
 os_image_api_version = 2
 tenant_id =
 os_novaclient_version = 2.1
+os_swiftclient_version = 2
```

Comment:
forgot this, and i fix in [new commit](https://github.com/cloudcomputinghust/CAL/pull/51/commits/553d97e731eb6f771eb862431ad0273d89736d4d)